### PR TITLE
Feature/contributors

### PR DESCRIPTION
**Pull request by TisRyno**
Rebuilds the contributors code to clean it up, improve the styling and fix the bugs it had.
Has basic mobile support but not focused on making it mobile friendly otherwise

### CODE REVIEW

**Comment by TisRyno** [comment]
Closes #3 
Thanks to @HargulS for the base code he provided.

File: backend/server.js [hunk 1]
```diff
@@ -1,13 +1,26 @@
...
 const app = express();
 
 const port = process.env.PORT || 5000;
 
-// Get route to confirm server is up when routing to /
-app.get("/", (req, res) => {
-  res.send({ express: "YOUR EXPRESS BACKEND IS CONNECTED TO REACT" });
-});
+app.use((req, res, next) => {
+  res.header('Access-Control-Allow-Origin', '*')
```
Comment (TisRyno) [inline backend/server.js:10]: Forgot to mention this is really bad but I'm lazy

File: src/components/Contributors.js [hunk 2]
```diff
@@ -1,59 +1,43 @@
...
+                            image={contributorsData.image}
+                            linkedIn={contributorsData.linkedIn}
+                            gitHub={contributorsData.gitHub}
+                            twitter={contributorsData.twitter}
+                        />
+
+                    )
+                })}
+            </div>
+            {usePrevious}
```
Comment (TisRyno) [inline src/components/Contributors.js:40]: I left this in but it feels redundant when there's no real state change?

File: src/components/Contributors.js [hunk 3]
```diff
@@ -1,59 +1,43 @@
...
 export default function Contributors(props) {
-
     const [contributorsData, setContributorsData] = useState();
 
-
-
     const getContributorsData = (e) => {
-       Axios.get(`http://localhost:5000/contributors/`)
-            .then((response) => {
+        Axios.get(`/api/contributors/`)
```
Comment (TisRyno) [inline src/components/Contributors.js:11]: Could just use
```js
import _contributors from "../data/contributors.json";
```
Would mean the API release can be pushed back

File: src/components/Contributors.js (hunk 2 above)
Comment (HargulS) [reply src/components/Contributors.js:40]: This was the only way I could find to stop the infinite loop.

**Pull request merged**